### PR TITLE
CentCom / NTC Changes by ChromeFoxxity - SPRITE CHANGES ONLY

### PR DESCRIPTION
**Pull request by IgiariValkyr**
## About The Pull Request

To explain this slightly better than ChromeFoxxity did, this PR edits all Centcom clothing to match the same color palette with darker greens and brighter golds. As well, this changes the turtlenecks of Centcom to have actual turtlenecks, rather than being simple ribbed sweaters. **THIS DOES NOT INCLUDE THE LOADOUT CHANGES OR ATTEMPTED FORCING OF A SWORD FOR REPRESENTITIVE.**

## Why It's Good For The Game

To quote ChromeFoxxity:
> The uniforms however, looked a little stale, the shading was a little off, from a dusty, green to a almost pitch black green instantly. This gives it a more vibrant coloring, while also keeping that classic green as the primary color. The gold has also been given more vibrancy, to make it look more like gold instead of just yellow stripes, it may look odd, but it fits more with the Captain's uniform, which these are based off of, or look similar to.

So the shorthand answer is visual consistency. All of the existing centcom uniforms use different shades of greens and golds for some reason, rather than being consistent like a company's uniform would be.

## Proof Of Testing

Built on my end, so whatever that's good for.
cmd_2024-12-09_15-57-51

<details>
<summary>Sprites of what changed</summary>

_**Commander's Uniform and Skirt**_
384160729-c9840989-6bad-4f72-bfda-52be30fce3d5

_**Official's Jumpsuit and Turtleneck**_
384160867-5903847c-980e-403c-a471-6190d1235ca2

_**ERT Officer's Turtleneck and Skirtleneck**_
384160958-ab55df7f-df69-4d5f-be62-b27733252af8

_**Centcom Replica Uniforms**_ (Notice the lack of belt and duller colors on the pants/skirt)
384161034-5997fd64-ebdc-4475-a7b1-11136faa336b

_**Additional Sprites**_
384161137-64fdab07-32c7-474f-9f83-b66050921ec0

</details>

## Changelog

:cl:
add: Adds different sprites to the CentCom uniforms, and clothing.
add: Adds a CentCom Carapace, and CentCom Gloves that aren't Naval.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/*
+*	Overwrites all the centcom icons with our own recolored versions; this means little to no mapping/spawning conflicts!
+*/
+
+/*
+* UNIFORMS
+*/
+/obj/item/clothing/under/rank/centcom/commander
+	name = "\improper CentCom commander's suit"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm:9]: No need to copy this on overrides if it isn't changed
```suggestion
```

File: modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+	desc = "An elegant uniform worn by CentCom's finest, comfortable trousers and gold marking denoting the rank of \"Commander\"."
+	icon = 'modular_zubbers/icons/obj/clothing/under/centcom.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/centcom.dmi'
+
+/obj/item/clothing/under/rank/centcom/centcom_skirt
+	name = "\improper CentCom commander's suitskirt"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm:15]: Ditto
```suggestion
/obj/item/clothing/under/rank/centcom/centcom_skirt
```

File: modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+/obj/item/clothing/under/rank/centcom/consultant/skirt
+	name = "\improper CentCom representative's suitskirt"
+	desc = "An elegant uniform worn by CentCom's station officials, comfortable skirt and silver marking denoting the rank of \"Representative\"."
+	icon = 'modular_zubbers/icons/obj/clothing/under/centcom.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/centcom.dmi'
+	icon_state = "rep_skirt"
+
+/obj/item/clothing/under/rank/centcom/official
+	name = "\improper CentCom official's suit"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm:35]: Ditto
```suggestion
/obj/item/clothing/under/rank/centcom/official
```

File: modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+	desc = "A formal suit worn by CentCom's paper pushers, a generic uniform usually used by Inspectors. The silver belt buckle denotes their rank with ease."
+	icon = 'modular_zubbers/icons/obj/clothing/under/centcom.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/under/centcom.dmi'
+
+/obj/item/clothing/under/rank/centcom/official/turtleneck
+	name = "\improper CentCom official's turtleneck"
+	desc = "A formal suit, with an added comfortable and warm turtleneck worn by CentCom's paper pushers, a generic uniform usually used by Inspectors. The silver belt buckle denotes their rank with ease."
+	icon_state = "official_turtleneck"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/clothing/clothing_overrides/centcom.dm:43]:
```suggestion
	icon_state = "official_turtleneck"// Sprites by Ebin-Halcyon
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Comment by IgiariValkyr** [comment]
Suggested changes pushed via fancy easy buttons here in github

**Review by Majkl-J — APPROVED** [review]

**Comment by ChromeFoxxity** [comment]
I just wanna say, the Representative's uniform, AKA the silver cap and uniform are unobtainable naturally anymore, I adding it to the garment bag.

**Comment by IgiariValkyr** [comment]
> I just wanna say, the Representative's uniform, AKA the silver cap and uniform are unobtainable naturally anymore, I suggest adding it to the garment bag.

Put them in the commandrobe.

**Pull request merged**